src/handlers/assign.rs

```diff
@@ -487,6 +487,22 @@ pub(super) async fn handle_command(
                     name.to_string()
                 } else {
                     let teams = crate::team_data::teams(&ctx.github).await?;
+                    // Determine if assignee is a team. If yes, add the corresponding label
+                    // team name here is without prefix 't-' (e.g. 'compiler', 'libs', etc.)
+                    if let Some(team) = teams.teams.get(&name) {
+                        let t_label = format!("t-{}", team.clone().name);
```

Comment:
```suggestion
                        let t_label = format!("t-{}", &team.name);
```
Minor nit, unless there's a reason to clone

Reply:
Just `team.name` should work btw